src/tools/rbd/action/Bench.cc

```diff
@@ -233,18 +264,30 @@ int do_bench_write(librbd::Image& image, uint64_t io_size,
   return 0;
 }
 
-void get_arguments(po::options_description *positional,
-                   po::options_description *options) {
+void get_arguments_for_write(po::options_description *positional,
+                             po::options_description *options) {
   at::add_image_spec_options(positional, options, at::ARGUMENT_MODIFIER_NONE);
   // TODO
   options->add_options()
-    ("io-size", po::value<Size>(), "write size (in B/K/M/G/T)")
+    ("io-size", po::value<Size>(), "op size (in B/K/M/G/T)")
     ("io-threads", po::value<uint32_t>(), "ios in flight")
-    ("io-total", po::value<Size>(), "total size to write (in B/K/M/G/T)")
-    ("io-pattern", po::value<IOPattern>(), "write pattern (rand or seq)");
+    ("io-total", po::value<Size>(), "total size for IO (in B/K/M/G/T)")
+    ("io-pattern", po::value<IOPattern>(), "io pattern (rand or seq)");
 }
```

Comment:
If you use it own get_arguments for bench-write, then I think it is better to leave options descriptions as they were before ('write' instead of 'IO')? Also, I would remove TODO comment in get_arguments_for_write (not sure what it means, but I think we don't want to improve this function in the future).